packages/edit-site/src/components/dataviews/in-filter.js

```diff
@@ -18,12 +19,16 @@ export default ( { filter, view, onChangeView } ) => {
 			? ''
 			: valueFound.value;
 
+	const id = useInstanceId( InFilter, 'dataviews__filters-in' );
+
 	return (
 		<SelectControl
+			id={ id }
 			value={ activeValue }
 			prefix={
 				<InputControlPrefixWrapper
-					as="span"
+					as="label"
+					htmlFor={ id }
 					className="dataviews__select-control-prefix"
 				>
 					{ filter.name + ':' }
```

Comment:
Aside: Just saw this, I think we should either remove the `:` or use sprint and __ here, not all locals support this notation I think.  cc @oandregal

Reply:
👍 This is going to become "Author is" when the new component is ready.

Reply:
I'll wait until this PR is merged to fix it – or if you'd rather do it yourself, @andrewhayward, you're welcome :)

Reply:
@oandregal I'll let you do that in a separate PR. Scope, etc!

Reply:
https://github.com/WordPress/gutenberg/pull/56075